index.src.html

```diff
@@ -586,6 +587,62 @@ <h3 id="feature-detect">Feature Detection</h3>
   Documents should not behave differently or not based on whether the UA
   implements this specification or not - all documents should assume it does.
 
+  <h3 id="premission-prompt">Permission Prompt</h3>
+
+  Following the discussions in
+  [[Issue#23](https://github.com/WICG/private-network-access/issues/23)], the
+  private network access permission prompt is introduced to relax mixed content
+  checks.
+
+  The goal of the permission is to allow communication from public websites to
+  local network servers over HTTP, which would otherwise be prevented by the
+  <a href="#secure-context-restriction">secure context restriction</a> and mixed
+  content checks. Migrating private network servers to HTTPS has indeed proven
+  to be often difficult, even sometimes impossible.
+
+  We propose adding a new parameter to the `fetch()` options bag. For example:
+  <pre highlight="js">
+    fetch("http://router.local/ping", {
+      targetAddressSpace: "private",
+    });
+  </pre>
+
+  This would instruct the browser to allow the fetch even though the scheme is
+  non-secure and obtain a connection to the target server. This ensures the
+  feature cannot be abused to bypass mixed content in general.
+
+  If the remote IP address does not belong to the IP address space specified as
+  the `targetAddressSpace` option value, then the request is failed. If it does
+  belong, then a CORS preflight request is sent as specified today. The target
```

Comment:
"today" does not make sense anymore in the context of a spec that defines said CORS preflight request.